amp-story/templates/ampmars/index.html

```diff
@@ -0,0 +1,967 @@
+        animation-play-state: running;
+        animation-timing-function: ease-out;
+      }
+
+      /*
+      * PAGE 03: WHAT TO BRING
+      * Stylelint requires `#trip-highlights` styles before `#what-to-bring`
+      */
+      #what-to-bring {
+        background-color: #0b0309;
+      }
+
+      #what-to-bring .suitcase-wrapper {
+        padding: 8vh 0;
+      }
+
+      #what-to-bring .suitcase-wrapper amp-img {
+        width: 100%;
+        position: absolute;
+      }
+
+      #what-to-bring[active] .mars-wrapper {
+        padding: 0;
+        align-content: flex-end;
+      }
+
+      #what-to-bring[active] .mars-wrapper amp-img {
+        animation-name: marsRising;
+        animation-duration: 1s;
+        animation-iteration-count: 1;
+        animation-direction: normal;
+        animation-fill-mode: forwards;
+        animation-play-state: running;
+        animation-timing-function: ease-out;
+      }
+
+      #what-to-bring .copy-layer {
+        align-content: flex-end;
+        padding: 0 19.6vw 9.7vh;
+      }
+
+      /* PAGE 05: ROVER TOURS */
+      #rover-tours {
+        background-color: #775444;
+      }
+
+      #rover-tours .button-container {
+        align-content: flex-end;
+        padding: 0 19.6vw 2.4vw;
+      }
+
+      #rover-tours .copy-layer {
+        padding: 15.6vw 21.5vw;
+      }
+
+      #rover-tours h2 {
+        margin-bottom: 0;
+      }
+
+      /* PAGE 05B: ROVER 360 */
+      #rover-360 {
+        background-color: #151515;
+      }
+
+      /* PAGE 06: LAST PAGE - WE CAN'T WAIT TO HAVE YOU ABOARD */
+      #last {
+        background-color: black;
+      }
+
+      #last .copy-layer {
+        padding: 12.4vw 24vw 10vw;
+        text-align: center;
+      }
+      #last h2 {
+        color: var(--yellow);
+        font-size: 8.6vw;
+        font-weight: 700;
+        font-family: 'Josefin Sans', sans-serif;
+        text-transform: uppercase;
+        display: flex;
+        flex-direction: column;
+      }
+
+      #last .last {
+        font-size: 7.2vw;
+      }
+      #last .italic {
+        color: #fff;
+        font-size: 5vw;
+        font-weight: 700;
+        font-style: italic;
+        font-family: 'Tienne', serif;
+        text-transform: initial;
+        margin-bottom: 1.5vw;
+      }
+
+      #last[active] .mars {
+        animation-name: endMars;
+        animation-duration: 30s;
+        animation-iteration-count: 1;
+        animation-direction: normal;
+        animation-fill-mode: forwards;
+        animation-play-state: running;
+        animation-timing-function: ease-out;
+      }
+
+      #last[active] .astronaut {
+        animation-name: endAstronaut;
+        animation-duration: 30s;
+        animation-iteration-count: 1;
+        animation-direction: normal;
+        animation-fill-mode: forwards;
+        animation-play-state: running;
+        animation-timing-function: ease-out;
+      }
+
+      @media only screen and (min-width: 48em) {
```

Comment:
Media queries are actually not very reliable with AMP stories; is there a reason this can't be done purely with responsive units?  Does the art direction actually change at larger screen sizes?  Can you provide screenshots at each of these breakpoints?

Reply:
The art direction doesn't change but there is an issue when using view units in large desktop as the browser calculates the entire browser screen which creates a disparity between mobile and desktop layout and font sizes.  Responsive units are actually creating the issue.

Reply:
Looking into it now.

Reply:
I've taken this over from christowiz and I'm seeing an issue with the responsive units as well. When using `vw` values, this does become a huge issue at the larger breakpoint when the story switches to a container fixed size, but the `vw` units still compute to full browser width. Looks like I need to do a decent amount of CSS rewriting to accommodate without breakpoints, but I'm unclear about the `amp-story-responsive-units` that you reference. Can you clarify what is supposed to happen with that? I'm not seeing any functional difference when I put in `AMP.toggleExperiment('amp-story-responsive-units', true)` in the console.

Reply:
If you put `AMP.toggleExperiment('amp-story-responsive-units', true)` in the console and then refresh the page, viewport units used in rules (not media queries) should get rewritten at runtime to make the viewport units be relative to the size of the `amp-story-page`, rather than the browser window.

Reply:
Hey @newmuis can you take a look at this and do a sanity check on your end? I had a lot of weird issues getting this to work in my browser, but it all the sudden started working for me, and seems good without the breakpoints now.